src/GafferScene/IECoreGLPreview/Renderer.cpp

```diff
@@ -1099,16 +1100,28 @@ class OpenGLRenderer final : public IECoreScenePreview::Renderer
 		{
 			const bool selected = parameter<bool>( parameters, "selection", false );
 
+			const PathMatcherData *omitted = nullptr;
+			IECore::CompoundDataMap::const_iterator it = parameters.find( "omitted" );
+			if( it != parameters.end() )
+			{
+				omitted = reportedCast<const PathMatcherData>( it->second.get(), "parameter", "omitted" );
+			}
```

Comment:
Any reason we can't use `parameter<PathMatcher>()` here instead? The PathMatcher copy operation is very cheap (basically just increments a reference count).

Reply:
Changed.